test/test_torch.py

```diff
@@ -1779,8 +1779,7 @@ def test_deterministic_cumsum(self, device):
             self.assertEqual(res0, res_cpu, atol=1e-3, rtol=1e-2)
 
     @onlyCUDA
-    @largeTensorTest('24GB', device='cuda')
-    @largeTensorTest('24GB', device='cpu')
+    @largeTensorTest('49GB')
```

Comment:
The original value here was just plain wrong; both `b` and `d` will exist until the end of the function, and it's pretty easy to verify that each is slightly more than 24GB in size.